src/style.css

```diff
@@ -0,0 +1,106 @@
+* {
+  box-sizing: border-box;
+}
+
+body,
+html {
+  margin: 0;
+  padding: 0 4px;
+}
+
+.serch_bar-container {
+  position: relative;
+  margin-top: 20px;
+  height: 70px;
+  border: 1px solid #e1e7ed;
+  box-shadow: 0 1px 8px rgba(61, 78, 97, 0.1);
+  border-radius: 4px;
+}
+
+.input_big {
+  height: 68px;
+  width: 100%;
+  vertical-align: middle;
+  padding-left: 62px;
+  font-family: "Avenir", sans-serif;
```

Comment:
Move it to the html selector and inherit it where the need

Reply:
done